src/utilities/sequence-editor/tree-utils.ts

```diff
@@ -86,6 +86,14 @@ export function* filterNodes(cursor: TreeCursor, filter?: (name: SyntaxNode) =>
   } while (cursor.next());
 }
 
+export function filterNodesToArray(cursor: TreeCursor, filter?: (node: SyntaxNode) => boolean): SyntaxNode[] {
+  return Array.from(filterNodes(cursor, filter));
+}
+
 export function nodeContents(input: string, node: SyntaxNode): string {
   return input.substring(node.from, node.to);
 }
+
+export function isDefined<Type>(maybeValue: Type | null | undefined): maybeValue is Type {
```

Comment:
This function already exists in `./src/utilities/generic.ts` as `filterEmpty()`

Reply:
I'll use that one. Should we should consider renaming that function like we're doing here? @duranb 

https://github.com/NASA-AMMOS/aerie/pull/1587